File: app/src/main/java/com/kite/mnemoai/data/network/Message.java
package com.kite.mnemoai.data.network;

public class Message{
    private String content;
    private String role;

    public Message(String content, String role) {
        this.content = content;
        this.role = role;
    }

    public String getContent() {
        return content;
    }

    public String getRole() {
        return role;
    }
}

File: core/network/src/main/java/com/kite/mnemoai/network/deepseek/model/Message.kt
package com.kite.mnemoai.network.deepseek.model

import com.google.gson.annotations.SerializedName

data class Message(
    val content: String,
    val role: Role,
    val name: String? = null
)

enum class Role {
    @SerializedName("system")
    SYSTEM,
    @SerializedName("user")
    USER,
    @SerializedName("assistant")
    ASSISTANT
}
